code/_onclick/click.dm

```diff
@@ -274,20 +274,20 @@
 	return
 
 /*
-	Middle click
+	Shift Middle click
 	Only used for swapping hands
 */
-/mob/proc/MiddleClickOn(atom/A)
+/mob/proc/ShiftMiddleClickOn(atom/A)
 	return
 
 /mob/living/carbon/MiddleClickOn(atom/A)
 	if(!src.stat && src.mind && src.mind.changeling && src.mind.changeling.chosen_sting && (iscarbon(A)) && (A != src))
 		next_click = world.time + 5
 		mind.changeling.chosen_sting.try_to_sting(src, A)
 	else
-		swap_hand()
+		..()

```

Comment:
So you don't randomly swap hands while you're trying to use a ling sting cause that's not supposed to be the button's function.